index.html

```diff
@@ -1,78 +1,72 @@
 <!DOCTYPE HTML>
 <html>
-
 <head>
-    <meta charset="UTF-8">
-    <title>LMHT</title>
-    <link type='text/css' href='css/style.css' rel='stylesheet' />
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" type="text/javascript"></script>
-    <script src="js/jquery-scrolltofixed-min.js" type="text/javascript"></script>
-    <link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />
-    <script type="text/javascript" src="js/stickynav.js"></script>
+  <meta charset="UTF-8">
+  <title>LMHT</title>
+  <link type='text/css' href='css/style.css' rel='stylesheet' />
+  <link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" type="text/javascript"></script>
+  <script type="text/javascript" src="js/stickynav.js"></script>
+  <script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+  <script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
+  <script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```

Comment:
You've still got duplicates:

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" type="text/javascript"></script>
<script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
```

```
<script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
<script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```

I also wonder if `<link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">` includes the base CSS or if it's just overrides for `<link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />`. Worth investigating.